build.sbt

```diff
@@ -1182,6 +1183,7 @@ lazy val `scio-examples`: Project = project
       "org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionCompatVersion,
       "org.slf4j" % "slf4j-api" % slf4jVersion,
       // runtime
+      "com.google.cloud.bigdataoss" % "gcs-connector" % s"hadoop2-$bigdataossVersion",
```

Comment:
@clairemcginty this used to be shipped with `scio-parquet`. I feel we should stay agnostic to the storage system and this must be manually added by users if needed now.